pkg/cluster/mpi/statefulset.go

```diff
@@ -209,14 +212,32 @@ func secretVolumes(cr *dcv1alpha1.MPICluster) ([]corev1.Volume, []corev1.VolumeM
 				},
 			},
 		},
+		{
```

Comment:
Would it be possible to only create the `ConfigMap`, `Volume`, `VolumeMount` if the keytab field in the CR is populated?